### PR TITLE
Update error message on swap too low quote test

### PR DESCRIPTION
**Pull request by gbecerra-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description
Error message priorities have been changed for the swap live app. This PR is about the expected error message when the amount is too big. The name of the TC is "Swap too low quote amounts from Bitcoin to Ethereum" and "Swap too low quote amounts from Ethereum to Bitcoin"

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts [hunk 1]
```diff
@@ -339,7 +339,7 @@ for (const { swap, xrayTicket } of tooLowAmountForQuoteSwaps) {
           await app.swap.verifySwapAmountErrorMessageIsDisplayed(
             electronApp,
             swap.accountToDebit,
-            "No quotes available.",
+            "Not enough balance, including network fee.",
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts:342]: if previous message isn't relevant anymore, I guess you can remove the last check all together (supposing our partner return quotes in **any** case)

Comment (gbecerra-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts:342]: No, partner will return no quotes in this case.

Comment (bharamboure-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts:342]: then my comment is relevant, you can delete next set of instructions all together

**Review by bharamboure-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts (hunk 1 above)
Comment (bharamboure-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts:342]: resolving, error priority updated on PTX level (LGTM to have this one now)

**Review by ypolishchuk-ledger — APPROVED** [review]

**Pull request merged**